benchmarking/macrobenchmark/src/main/java/com/example/macrobenchmark/ExampleStartupBenchmark.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 The Android Open Source Project
```

Comment:
2023

Reply:
These were created in 2022. No need to update the copyright year.